scanpipe/pipes/scancode.py

```diff
@@ -481,10 +481,10 @@ def make_results_summary(project, scan_results_location):
         key_files.append(resource_data)
 
         packages = resource.discovered_packages.all()
-        packages_data = [
-            DiscoveredPackageSerializer(package).data for package in packages
-        ]
-        key_files_packages.extend(packages_data)
+        for package in packages:
+            package_data = DiscoveredPackageSerializer(package).data
+            if package_data not in key_files_packages:
```

Comment:
It seems a bit much to test on a whole data structure such as `package_data`.
Could we use a more specific id instead?